src/__tests__/query-decomposer.spec.ts

```
@@ -0,0 +1,24 @@
+import { QueryDecomposerService } from '../resources/rag/query-decomposer.service';
+
+// Minimal mock for ChatGroq
+class MockLLM {
+  async invoke(messages: any) {
```

ESLint

[error] 'messages' is defined but never used.

(@typescript-eslint/no-unused-vars)